extensions/amp-analytics/0.1/vendors.js

```diff
@@ -1146,6 +1146,46 @@ export const ANALYTICS_CONFIG = /** @type {!JSONType} */ ({
       'image': true,
     },
   },
+
+  'ibeatanalytics': {
+    'requests': {
+      'host': 'https://ibeats.indiatimes.com',
+      'base': 'https://ibeats.indiatimes.com/iBeat/pageTrendlogAmp.html',
+      'pageview': '${base}?' +
+                '&h=ibeats.indiatimes.com' +
+                '&d=ibeats.indiatimes.com' +
+                '&url=${url}' +
+                '&k=${key}' +
+                '&ts=${time}' +
+                '&ch=${channel}' +
+                '&sid=${uid}' +
+                '&at=${agentType}' +
+                '&ref=${documentReferrer}' +
+                '&aid=${aid}' +
+                '&loc=1' +
+                '&ct=1' +
+                '&cat=${cat}' +
+                '&scat=${scat}' +
+                '&ac=1' +
+                '&tg=${tags}' +
+                '&ctids=${catIds}' +
+                '&pts=${pagePublishTime}' +
+                '&auth=${author}' +
+                '&pos=${position}' +
+                '&iBeatField=${ibeatFields}',
+    },
+    'triggers': {
+      'defaultPageview': {
+        'on': 'visible',
+        'request': 'pageview',
+      },
+    },
+    'transport': {
+      'beacon': false,
+      'xhrpost': false,
+      'image': true,
+    },
```

Comment:
Hi @lannka , yes it is just copy paste from other sample,
ibeat simply receive post request with data mentioned in var tag and return 1 pixel as response.
Please suggest me the correct form of above tag for ibeat